### PR TITLE
CU-2vee8bh - Register page

### PR DESCRIPTION
**Pull request by d-wasilewski**

### CODE REVIEW

**Comment by pmakowski23** [comment]
Task linked: [CU-2vee8bh Register Page](https://app.clickup.com/t/2vee8bh)

File: src/pages/index.astro [hunk 1]
```diff
@@ -9,6 +9,8 @@ import CallToAction from 'components/index/CallToAction.astro';
 ---
 
 <Layout title="DELTA">
+  <RegisterForm client:only="solid-js"/>
```
Comment (pmakowski23) [inline src/pages/index.astro]: Please render it on separate page and add button in header that will lead to registration.

**Review by pmakowski23 — CHANGES_REQUESTED** [review]
Please improve responsiveness

**Review by pmakowski23 — APPROVED** [review]
Good job!

**Review by kwawrzynczak — APPROVED** [review]

**Pull request merged**